src/components/Footer.js

```diff
@@ -11,6 +12,12 @@ const FooterContainer = styled.div`
   display: flex;
   justify-content: space-between;
   padding: 50px 0 80px 0;
+
+  @media only screen and (max-width: 768px) {
+    display: flex;
```

Comment:
No need to repeat these styles. If they’re at the base level, they’ll be present at all break points.